src/backend/joanie/core/api/client/__init__.py

```diff
@@ -541,6 +542,59 @@ def withdraw(self, request, pk=None):  # pylint: disable=no-self-use, invalid-na
 
         return Response(status=HTTPStatus.NO_CONTENT)
 
+    @extend_schema(
+        request=None,
+        responses={
+            (200, "application/json"): OpenApiTypes.OBJECT,
+            404: serializers.ErrorResponseSerializer,
+            422: serializers.ErrorResponseSerializer,
+        },
+    )
+    @action(detail=True, methods=["POST"])
+    def submit_installment_payment(self, request, pk=None):  # pylint: disable=unused-argument
+        """
+        Pay a failed installment that was scheduled for a given order.
+        """
+        order = self.get_object()
+        if order.state != enums.ORDER_STATE_FAILED_PAYMENT:
+            return Response(
+                {
+                    "detail": "The order is not in failed payment state for last payment."
```

Comment:
I do not understand well this error message.

```suggestion
                    "detail": "The order is not in failed payment state."
```